pom.xml

```diff
@@ -63,11 +63,20 @@
 
 	<build>
 		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-surefire-plugin</artifactId>
-				<version>2.22.1</version>
-			</plugin>
+<!--			<plugin>-->
+<!--				<groupId>org.apache.maven.plugins</groupId>-->
+<!--				<artifactId>maven-surefire-plugin</artifactId>-->
+<!--				<version>2.22.1</version>-->
+<!--			</plugin>-->
```

Comment:
Go ahead and remove this code completely, we have the git history to restore it if need be!

Reply:
Yes sir!